packages/components/managed-data-table/src/data-table-settings/data-table-settings.spec.js

```diff
@@ -146,11 +147,11 @@ describe('DataTableSettings', () => {
         "[name='table-settings-dropdown']"
       );
 
-      fireEvent.focus(dropDown);
-      fireEvent.keyDown(dropDown, { key: 'ArrowDown' });
-      fireEvent.keyUp(dropDown, { key: 'ArrowDown' });
-      fireEvent.keyDown(dropDown, { key: 'Enter' });
-      fireEvent.keyUp(dropDown, { key: 'Enter' });
+      await fireEvent.focus(dropDown);
+      await fireEvent.keyDown(dropDown, { key: 'ArrowDown' });
+      await fireEvent.keyUp(dropDown, { key: 'ArrowDown' });
+      await fireEvent.keyDown(dropDown, { key: 'Enter' });
+      await fireEvent.keyUp(dropDown, { key: 'Enter' });
```

Comment:
Sorry I didn't mean to do it like this. The fire event is not async, the expectation AFTER firing the event MIGHT need to be async.

For example, if you're asserting some element or text after some user interactions, you can use `await findBy` instead of `expect().toBeInTheDocument`.

Again, it might not be necessary but you should be aware that after firing some events the expectation could be async, otherwise you might get `act` warnings.

Reply:
I've removed the `async`. Currently I'm not getting any warnings